packages/mantine/src/components/table/table-actions/TableActionItem.tsx

```diff
@@ -1,12 +1,17 @@
-import {BoxProps, CompoundStylesApiProps, Menu, PolymorphicFactory, polymorphicFactory, useProps} from '@mantine/core';
+import {CompoundStylesApiProps, Menu, PolymorphicFactory, polymorphicFactory, useProps} from '@mantine/core';
 import {ReactNode} from 'react';
-import {Button} from '../../button';
+import {Button, ButtonProps} from '../../button';
 import {useTableContext} from '../TableContext';
 import {useTableActionContext} from './TableActionContext';
 
 export type TableActionItemStylesNames = 'actionItemRoot';
 
-export interface TableActionItemProps extends BoxProps, CompoundStylesApiProps<TableActionItemFactory> {
+export interface TableActionItemProps
+    extends Omit<
+            ButtonProps,
+            'classNames' | 'styles' | 'vars' | 'variant' | 'leftSection' | 'rightSection' | 'disabled'
```

Comment:
Since we now extend ButtonProps we can remove `rightSection` and `disabled` from the interface and from the Omit. It will use Mantine's type

Reply:
I agree, I kept it because our JSDoc gave a different definition, but after I looked at it it's pretty much the same :P